hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/state/ProxyWorldUpdater.java

```diff
@@ -354,7 +362,12 @@ public void commit() {
      */
     @Override
     public @NonNull ProxyWorldUpdater updater() {
-        return new ProxyWorldUpdater(hederaOperations.begin(), evmFrameStateFactory, this);
+        final var child = new ProxyWorldUpdater(hederaOperations.begin(), evmFrameStateFactory, this);
+        // "Hand off" the pending creation to the child, if there is one
```

Comment:
Can you please explain this?  How and why does the child need to act on pending creations?